trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    name = @nameable.correct_name
+    name.length > max ? name[0, 10] : name
+  end
```

Comment:
- [ ] There is an error raised when testing the trimmer decorator, please fix that.

![image](https://github.com/danutibe07/School_Library/assets/68796232/ec1ed053-1b62-439f-bdf3-41199d61e4f9)

[Hint] You can do it like that:
```ruby
def correct_name
    name = @nameable.correct_name[0..9]
end
```

Reply:
well resolved! ✔️